src/libraries/System.Data.Common/src/System/Data/SQLTypes/SQLBinary.cs

```diff
@@ -17,7 +17,7 @@ public struct SqlBinary : INullable, IComparable, IXmlSerializable, IEquatable<S
         // NOTE: If any instance fields change, update SqlTypeWorkarounds type in System.Data.SqlClient.
         private byte[]? _value;
 
-        private SqlBinary(bool fNull)
+        private SqlBinary(bool _)
```

Comment:
I don't understand the purpose of this ctor.  Shouldn't the call site just use `default`?